Tutorial rework - Blockchain (#2443)
timescale/docs: tutorials/blockchain-analyze/index.md

---
title: Analyze the Bitcoin blockchain
excerpt: Analyze the Bitcoin blockchain
products: [cloud]
keywords: [intermediate, crypto, blockchain, Bitcoin, finance, analytics]
layout_components: [next_prev_large]
content_group: Analyze the Bitcoin blockchain
---

# Analyze the Bitcoin blockchain

[Blockchains][blockchain-def] are, at their essence, a distributed database. The
[transactions][transactions-def] in a blockchain are an example of time-series
data. You can use Timescale to query transactions on a blockchain, in exactly the
same way as you might query time-series transactions in any other database.

In this tutorial, you use Timescale hyperfunctions to analyze transactions
on the Bitcoin blockchain. You can use these skills to query any data on a
blockchain, including other cryptocurrencies, smart contracts, or health data.

## Prerequisites

Before you begin, make sure you have:

*   Signed up for a [free Timescale account][cloud-install].

> *   <Optional />If you want to graph your queries, signed up for a
>     [Grafana account][grafana-setup].

Reviewer: ```suggestion
*   <Optional />Signed up for a [Grafana account][grafana-setup] to graph your queries.
```

## Steps in this tutorial

This tutorial covers:

1.  [Set up up your dataset][blockchain-dataset]
1.  [Analyze your dataset][blockchain-query]

## About analyzing the Bitcoin blockchain with Timescale

This tutorial uses a sample Bitcoin dataset to show you how to aggregate
blockchain transaction data, and construct queries to analyze information from

> the aggregations. The queries you do in this tutorial are used to do things like
> determine if a cryptocurrency has a high transaction fee, shows any correlation
> between transaction volumes and fees, or if it's expensive to mine.

Reviewer: ```suggestion
the aggregations. The queries in this tutorial help you
determine if a cryptocurrency has a high transaction fee, shows any correlation
between transaction volumes and fees, or if it's expensive to mine.
```
